src/server/components/formRunner/lawyers.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Country,
+  LawyerListItemGetObject,
+  LawyerListItemJsonData,
+  ServiceType
+} from "server/models/types";
+import { FormRunnerFields, FormRunnerQuestion } from "server/components/formRunner/types";
+import { parseJsonFormData } from "server/components/formRunner/helpers";
+
+export async function generateFormRunnerWebhookData(listItem: LawyerListItemGetObject,
+                                                    listCountry?: Partial<Country>,
+                                                    isUnderTest?: boolean): Promise<Array<Partial<FormRunnerQuestion>>> {
+  const questions = await parseJsonFormData(ServiceType.lawyers, isUnderTest);
+
+  questions.forEach((question) => {
+    question.fields?.forEach((field) => {
+      const jsonData: LawyerListItemJsonData = listItem.jsonData ?? {};
+
+      switch (field.key) {
+        case FormRunnerFields.country :
+          field.answer = ((listCountry?.name) != null) ? listCountry.name : "";
+          break;
+        case FormRunnerFields.addressCountry :
+          field.answer = listItem.address.country.name;
+          break;
+        case FormRunnerFields.addressLine1 :
+          field.answer = listItem.address.firstLine;
+          break;
+        case FormRunnerFields.addressLine2 :
+          field.answer = listItem.address.secondLine ?? "";
+          break;
+        case FormRunnerFields.city :
+          field.answer = listItem.address.city;
+          break;
+        case FormRunnerFields.postcode :
+          field.answer = listItem.address.postCode;
+          break;
+        case FormRunnerFields.phoneNumber :
+          field.answer = jsonData.phoneNumber;
+          break;
+        case FormRunnerFields.emergencyPhoneNumber :
+          field.answer = jsonData.emergencyPhoneNumber ?? "";
```

Comment:
some of these are optional fields in the db. when the user resubmits their data, will these be `""` instead of null/undefined? will they get replaced?

Reply:
Currently, an unspecified field does not get persisted so there are instances where it would be undefined.

Reply:
as above, this code is now redundant as it's replaced with the lodash `.get()` implementation.